### PR TITLE
verifying error messages while signing in

### PR DESCRIPTION
**Pull request by KaranTikellu7**

### CODE REVIEW

File: tests/verify_error_messages_while_sigining_in.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+describe('Verifying error messages While signing in', ()=>
+{    
+    it('With Blank Email Address and Valid Password',()=>
+    {
+        Signin_Page.Password_Textbox.setValue(Common.Valid_Password)
+        Signin_Page.Login_Button.click()
+
+        Common.Wait(1000)
+        expect(Common.ErrorMessage.isVisible()).to.true
+        expect(Common.ErrorMessage.getText()).to.equal(Common.Invalid_ErrorMessage)
```
Comment (prabhakar-vineti) [inline tests/verify_error_messages_while_sigining_in.js]: When you are verifying text for the `Error Message` it means that it is visible on page.  There is no requirement to verify that again on line no.17.

**Comment by prabhakar-vineti** [comment]
@KaranTikellu7 

Please update.

File: tests/verify_error_messages_while_sigining_in.js (hunk 1 above)
Comment (prabhakarbattula) [inline tests/verify_error_messages_while_sigining_in.js:17]: When you are verifying text for the Error Message it means that it is visible on page. There is no requirement to verify that again on line no.17.

**Comment by prabhakarbattula** [comment]
@KaranTikellu7 

Please update.

**Pull request merged**